internal/benchmarks/src/transaction-results.bench.ts

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import { Wallet } from 'fuels';
+import type { TransferParams } from 'fuels';
+import { launchTestNode, TestAssetId } from 'fuels/test-utils';
+import { bench } from 'vitest';
+
+/**
+ * @group node
+ */
+describe('Transaction Submission Benchmarks', () => {
+  bench('should successfully transfer a single asset between wallets', async () => {
+    using launched = await launchTestNode();
```

Comment:
@maschad For transaction submissions I believe it is better to set up the node with the following:

```ts
 args: ['--poa-instant', 'false', '--poa-interval-period', '1s'],
``` 

This will give us results closer to reality.

Reply:
There will be three environments which these tests will run in - local, devnet and testnet.  Running them in the devnet and testnet environments will give us results _closer_ to reality, so no need to introduce an artificial delay.